packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -57,6 +57,7 @@ public boolean shouldOverrideUrlLoading(
               return true;
             }
 
+            @SuppressWarnings("deprecation")
```

Comment:
Why not fix the problem rather than hiding it? This is a global statement for this entire pull request